analytics_data_api/v0/serializers.py

```diff
@@ -241,3 +241,31 @@ class Meta(object):
         model = models.CourseActivityWeekly
         # TODO: Add 'posted_forum' here to restore forum data
         fields = ('interval_start', 'interval_end', 'course_id', 'any', 'attempted_problem', 'played_video', 'created')
+
+
+class VideoSerializer(ModelSerializerWithCreatedField):
+    # pipeline_video_id is renamed to video_id
```

Comment:
Now that you mention it, I think you're right.  I was thinking that if the video_id changed formats (maybe we find the edx video IDs are available), we could just leave the field as is and swap out the logic.

I do think it's probably better to be specific about where the ID is coming from though--less confusing!  I'll update the PR in a bit.